pom.xml

```diff
@@ -490,6 +490,11 @@
   </distributionManagement>
   <dependencyManagement>
     <dependencies>
+      <dependency>
```

Comment:
Could you add a brief comment here wrt why this is needed -- it sounds like it is to override version of a transitive dependency (protobuf generator?); would be good to know wrt future increases of version.